infrastructure/templates/public-api/application/etlFunctionApp.bicep

```diff
@@ -0,0 +1,140 @@
+@description('Specifies the Subscription to be used.')
+param subscription string
+
+@description('Specifies the location for all resources.')
+param location string
+
+//Specific parameters for the resources
+@description('Application : Insights name')
+param applicationInsightsName string
+
+@description('Function App Plan operating system')
+@allowed([
+  'Windows'
```

Comment:
Why don't we just choose Windows?

Reply:
Linux is smaller, faster and cheaper than running a windows vm

Reply:
Have we chosen which one we want to use for the public API project? Just wondered why we'd want 2 options rather than choosing just 1?